source/timemory/components/rusage/backends.hpp

```diff
@@ -57,10 +57,9 @@
 #    if !defined(WIN32_LEAN_AND_MEAN)
 #        define WIN32_LEAN_AND_MEAN
 #    endif
-// currently, this is causing a bunch of errors, need to disable
-// #    include <psapi.h>
-#    include <stdio.h>
 #    include <windows.h>
+#    include <psapi.h>
```

Comment:
Add a `//` here, e.g.

```cpp
#    include <stdio.h>
#    include <windows.h>
//
#    include <psapi.h>
```

My clang-format settings sort the headers alphabetically

Reply:
I added the comment. But a blank line would also work.